examples/00-EDB/10_GDS_workflow.py

```diff
@@ -16,39 +15,40 @@
 from pyaedt.edb_core.edb_data.control_file import ControlFile
 
 ###############################################################################
-# Download file
-# ~~~~~~~~~~~~~
-# Download the AEDB file and copy it in the temporary folder.
-temppath = tempfile.gettempdir()
-local_path = pyaedt.downloads.download_file('gds')
-c_file_in = os.path.join(
-    local_path, "sky130_fictitious_dtc_example_control_no_map.xml"
-)
-c_map = os.path.join(local_path, "dummy_layermap.map")
-gds_in = os.path.join(local_path, "sky130_fictitious_dtc_example.gds")
-gds_out = os.path.join(temppath, "example.gds")
+# ## Fetch Example Data
+#
+# Download the EDB folder and copy it to a temporary folder.
+# The following files are used in this example:
+# - _sky130_fictious_dtc_exmple_contol_no_map.xml_
+#   defines physical information such
+#   as material properties, stackup layers, and boundary conditions.
+# - _dummy_layermap.map_
+#   maps properties to stackup layers.
```

Comment:
```suggestion
# ## Fetch Example Data
#
# Download the EDB folder and copy it to a temporary folder.
# The following files are used in this example:

# - _sky130_fictious_dtc_exmple_contol_no_map.xml_:
#   Defines physical information such
#   as material properties, stackup layers, and boundary conditions.

# - _dummy_layermap.map_:
#   Maps properties to stackup layers.
```
I know that a blank line is needed before the first item in a list and a last item in a list to render correctly in the documentation. However, I'm not sure if this is the case in these Jupyter notebook examples. I made a similar edit earlier.